.github/workflows/build_and_release_gem.yml

```diff
@@ -92,3 +98,45 @@ jobs:
           sudo apt-get update && sudo apt-get install -y ruby-dev g++ make
           gem install solarwinds_apm --version ${{ steps.build.outputs.gem_version }}
           ruby ./scripts/test_install.rb
+
+  # extract the built layer from artifacts, then scan it with reverselab
+  reverselab_scan_gem:
+    needs:
+      - publish_to_ruby_gem
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: extract current solarwinds_apm version
```

Comment:
Minor, seems you should be able to [pass the version info](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs) from previous job to this one.

Reply:
Updated for github package and rubygems